Configuration.cs

```diff
@@ -0,0 +1,12 @@
+namespace DotNetMaui;
+
+public static class Configuration
+{
+
+    public const string SentryDsn = "https://ec46b4c5e07cd8e976033a8a1e63cb01@o87286.ingest.us.sentry.io/4509153190739968";
+    //public const string SentryDsn = "https://eb18e953812b41c3aeb042e666fd3b5c@o447951.ingest.sentry.io/5428537";
+    
+    public const string BackendUrl = "https://laravel.empower-plant.com";
```

Comment:
@realkosty will we get a `aspnetcore` endpoint?

Reply:
yes we do, http://aspnetcore.empower-plant.com/products